### PR TITLE
Add `PageLayoutTrait` with Main and sidebar method

### PR DESCRIPTION
**Pull request by amitaibu**
Adding a helper method for the main and sidebar page layout

image

### CODE REVIEW

File: web/modules/custom/server_general/src/PageLayoutTrait.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\server_general;
+
+/**
+ * Helper methods to build a main and sidebar layout.
```
Comment (amitaibu) [inline web/modules/custom/server_general/src/PageLayoutTrait.php:8]:
```suggestion
 * Helper methods to build Page layouts.
```

File: web/modules/custom/server_general/src/PageLayoutTrait.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+declare(strict_types=1);
+
+namespace Drupal\server_general;
+
+/**
+ * Helper methods to build Page layouts.
+ *
+ * A regular single column page layout doesn't need any method here, as one can
+ *  use inside PEVB something like`$this->wrapContainerWide($elements)`.
```
Comment (amitaibu) [inline web/modules/custom/server_general/src/PageLayoutTrait.php:11]:
```suggestion
 * use inside PEVB something like`$this->wrapContainerWide($elements)`.
```

File: web/modules/custom/server_general/src/PageLayoutTrait.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ * Helper methods to build Page layouts.
+ *
+ * A regular single column page layout doesn't need any method here, as one can
+ * use inside PEVB something like`$this->wrapContainerWide($elements)`.
+ * So it's likely this trait will hold only the Main and sidebar helper method,
+ * unless there's a need for a more complex layout.
+ *
+ * Trait is providing helper methods for each card. One method equals one theme
```
Comment (amitaibu) [inline web/modules/custom/server_general/src/PageLayoutTrait.php:15]:
```suggestion
 * Trait is providing helper methods for each layout. One method equals one theme
```

**Review by andreytroeglazov — APPROVED** [review]

**Comment by andreytroeglazov** [comment]
I have already tested it :slightly_smiling_face:

**Pull request merged**